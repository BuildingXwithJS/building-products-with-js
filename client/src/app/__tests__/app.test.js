import App from '../index';

test('# App', () => {
  const wrapper = shallow(
    <App>test</App>
  );
  expect(wrapper).toMatchSnapshot();
});
