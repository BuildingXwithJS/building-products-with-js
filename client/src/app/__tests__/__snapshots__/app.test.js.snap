exports[`test # App 1`] = `
<div
  className="container">
  test
  <Component />
</div>
`;
